core/src/components/item/item.ionic.scss

```diff
@@ -21,6 +22,7 @@
   --padding-start: #{globals.$ionic-space-400};
 
   font-family: globals.$ionic-font-family;
+  font-size: globals.$ionic-font-size-400;
```

Comment:
Should this use `body/lg/medium` instead?

Reply:
https://github.com/ionic-team/ionic-framework/pull/29905/commits/1c66e25c1790d41124cb094c4cbbf3529012af0e